sympy/categories/diagram_drawing.py

```diff
@@ -2155,14 +2155,14 @@ def _check_free_space_vertical(dom_i, cod_i, dom_j, grid):
         if dom_j == 0:
             free_left = True
         else:
-            free_left = all([not grid[i, dom_j - 1] for i in
-                             range(start, end + 1)])
+            free_left = all(not grid[i, dom_j - 1] for i in
+                            range(start, end + 1))
 
         if dom_j == grid.width - 1:
             free_right = True
         else:
-            free_right = all([not grid[i, dom_j + 1] for i in
-                              range(start, end + 1)])
+            free_right = all(not grid[i, dom_j + 1] for i in
+                             range(start, end + 1))
```

Comment:
I suppose it is probably faster to do `not any` rather than `all not` because only one negation is needed.

Reply:
You are probably correct. I have not considered that type of changes, but can have a quick look.